nvflare/private/fed/utils/app_deployer.py

```diff
@@ -25,49 +26,47 @@
 from .app_authz import AppAuthzService
 
 
-class AppDeployer(object):
-    def __init__(self, workspace: Workspace, job_id: str, job_meta: dict, app_name: str, app_data):
-        self.app_name = app_name
-        self.workspace = workspace
-        self.app_data = app_data
-        self.job_id = job_id
-        self.job_meta = job_meta
+class AppDeployer(AppDeployerSpec):
+    def __init__(self):
+        AppDeployerSpec.__init__(self)
```

Comment:
We need to make sure those instance variables (self.workspace, self.app_data, self.job_id, ...) is not used in all other places

For example if some other code has
```
app_deployer = AppDeployer(XXX)
app_deployer.job_id
```